randovania/games/super_metroid/json_data/Wrecked Ship.txt

```diff
@@ -148,8 +143,9 @@ Extra - asset_id: 54
   * Super Missile Door to Basement/Door to Wrecked Ship Main Shaft
   > Door to Wrecked Ship West Super Room
       All of the following:
-          Morph Ball
-          Morph Ball Bombs or Power Bombs or Spring Ball
+          Can Lay Any Bombs
```

Comment:
These blocks can be broken with Wave Beam as well.

Reply:
Only with wave, or other beams too?

Reply:
Just tested it, wave and spazer can get it. The other beams are too narrow.

Reply:
I'll put beams as intermediate knowledge if you dont mind